docs/source/user-guide/index.rst

```diff
@@ -52,4 +52,5 @@ Integrations
     integrations/spark_databricks
     integrations/pytest
     integrations/h2o
-    integrations/airflow
\ No newline at end of file
+    integrations/airflow
+    general/exporting_results/examples/plot_exports_output_to_wandb
```

Comment:
I see we can combine sphinx galler and rst files in same hierarchy? (good news)

Shouldn't it also be deleted from the other toc...?

Reply:
![image](https://user-images.githubusercontent.com/20860465/168581936-780e9595-dc91-4f9a-84cc-63b937269013.png)

Reply:
You like it better like this? (feels a bit weird to have it double-pointed, would put it only in integrations...)